src/components/dataDisplays/CollapsibleRow.jsx

```diff
@@ -57,7 +66,15 @@ export default function CollabsibleRow({
                 ...cellStyles,
               }}
             >
-              {cellRenderer(cellValue, datum)}
+              {customCellRenderer ? (
+                customCellRenderer(cellValue, datum)
+              ) : (
+                <RequestedCellRenderer
```

Comment:
There's something that I might not understand here. How is this a component??

Reply:
Nvm. I get it now. The renderers imported from the renderers directory are the components.